packages/x-date-pickers-pro/src/MultiInputDateRangeField/MultiInputDateRangeField.types.ts

```diff
@@ -65,7 +67,7 @@ export interface MultiInputDateRangeFieldSlots {
   /**
    * Form control with an input to render a date.
    * It is rendered twice: once for the start date and once for the end date.
-   * @default TextField from '@mui/material' or PickersTextField if `enableAccessibleFieldDOMStructure` is `true`.
+   * @default <PickersTextField />, or <TextField /> from '@mui/material' if `enableAccessibleFieldDOMStructure` is `false`.
```

Comment:
should we wrap these in backticks? Not sure if we do this and if we are consistent with that.
```suggestion
   * @default `<PickersTextField />`, or `<TextField />` from '@mui/material' if `enableAccessibleFieldDOMStructure` is `false`.
```